### ISSUE TITLE
Relying on old offices such as HTML

### ISSUE THREAD

**Issue by abd-mtk** [jonataslaw/get_storage#151]
Hi, is there an upcoming update that will eliminate the dependence on HTML? I am excited to use it on Web Assembly.Relying on old offices such as HTML

**Comment by jonataslaw** [comment]
Yes, of course. I created two new libraries (Get_mutex and Get_packer) that will be responsible for serialization and deserialization, as well as writing binary to disk (we will abandon json and localstorage for web to have something closer to a database, but still lightweight and MUCH more performant). This will allow wasm compilation as well.

**Comment by bybabek** [comment]
> Yes, of course. I created two new libraries (Get_mutex and Get_packer) that will be responsible for serialization and deserialization, as well as writing binary to disk (we will abandon json and localstorage for web to have something closer to a database, but still lightweight and MUCH more performant). This will allow wasm compilation as well.

Do you have any estimated time frame for when these updates will be available?

**Comment by jonataslaw** [comment]
Today, the library is done, but I need put a migrate function because we are change the way how the data is being stored

**Comment by darkfrog26** [comment]
@jonataslaw your response is a bit ambiguous. Is it possible to use this now or are we still waiting for a migration?

**Comment by Ssiswent** [comment]
> Today, the library is done, but I need put a migrate function because we are change the way how the data is being stored今天，库已经完成，但我需要添加一个迁移函数，因为我们正在改变数据的存储方式

Just checking in on the progress since it's been a month. I appreciate all the hard work you're putting into this. Thank you!

**Comment by darkfrog26** [comment]
@Ssiswent I just migrated to `shared_preferences` with very little change to my code, and it now works on WASM.

**Comment by antiv** [comment]
HI, do you have infomation, when tis will be available?

**Comment by Joseph-Nathan** [comment]
any news !?
